src/BinDeps.jl

```diff
@@ -561,6 +561,21 @@ function eval_anon_module(context, file)
     return
 end
 
+"""
+    glibc_version()
+
+For Linux-based systems, return the version of glibc in use. For non-glibc Linux and
+other platforms, returns `nothing`.
+"""
+function glibc_version()
+    Compat.Sys.islinux() || return
+    libc = ccall(:jl_dlopen, Ptr{Void}, (Ptr{Void}, UInt32), C_NULL, 0)
```

Comment:
Ah, so passing `C_NULL` to `Libdl.dlopen()` doesn't work anymore, but we can still `ccall(:jl_dlopen)` with `C_NULL` to get this behavior?  This seems kind of underhanded.  Is this a sign that we should have a `libc_name` in `Base` a la `libblas_name`?

Reply:
Perhaps worth opening an issue about. Seems useful.

Reply:
i don't think it was ever allowed but I do think it's useful.